packages/veritone-react-common/src/components/MediaInfoPanel/EditMetadataDialog/index.js

```diff
@@ -0,0 +1,218 @@
+import React, { Component } from 'react';
+import Button from 'material-ui/Button';
+import IconButton from 'material-ui/IconButton';
+import Icon from 'material-ui/Icon';
+import Dialog, {
+  DialogActions,
+  DialogContent,
+  DialogTitle
+} from 'material-ui/Dialog';
+import TextField from 'material-ui/TextField';
+import { objectOf, func, bool, any } from 'prop-types';
+import { get } from 'lodash';
+import { withStyles } from 'material-ui/styles';
+import withMuiThemeProvider from '../../../helpers/withMuiThemeProvider';
+import styles from './styles.scss';
+
+@withMuiThemeProvider
+class EditMetadataDialog extends Component {
+  static propTypes = {
+    metadata: objectOf(any),
+    isOpen: bool,
+    onSave: func,
+    onClose: func
+  };
+
+  state = {
+    filename: get(this.props.metadata, 'veritoneFile.filename', ''),
+    source: get(this.props.metadata, 'veritoneCustom.source', ''),
+    programLiveImage: get(this.props.metadata, 'veritoneProgram.programLiveImage', ''),
```

Comment:
is there an option to use the signed version of programLiveImage/programImage?
signed image to display and unsigned when saving?

btw, we should also pass the signed image into the media player to display

Reply:
yeah, will do.
Will take into account setting signed image to the player. I don't think we set image do this right now in this MDP implementation

Reply:
Btw just tried signed uri and got 403 access denied - missing some auth stuff.
I think it's ok to go as it is right now
I will fix this 403 for signed uri sometime this week